docs/DeveloperGuide.md

```diff
@@ -310,7 +310,7 @@ Use case ends
 **MSS**
 1. User enters a command to edit all contacts with specific tags to new tag.
 2. Cher will change all contacts containing specified tags to the new tag.
-3. Cher will show the contacts that have been changed.<br>
+3. Cher will show the contacts that have the new tag.<br>
```

Comment:
UC5 & UC7 are missing Cher save to disk

Reply:
Also for UC5 and UC7 Extension, I think it's 2a. Cher detects error in input?

Reply:
Also may be more specific to write Cher will show all contacts that have the new tag

Reply:
I believe its 1a, Cher detects error in input as the error is caused by 1a

Reply:
1a seems right since the validation is done right after the command input is entered